### ISSUE TITLE
Documentation for example datasets

### ISSUE THREAD

**Issue by VeckoTheGecko** [OceanParcels/oceanparcels_website#101]
Is there any documentation about the datasets in `examples-data` (which are in turn used throughout the parcels examples)? Perhaps it would be worth adding a file `examples-data/README.md` to add quick descriptions to these. This can be propagated through to users in parcels if we want as well.

**Comment by erikvansebille** [comment]
Yes, I fully agree! Probably best if I write this README, since I know where the datasets came from. I'll assign to myself
